common/djangoapps/static_replace/__init__.py

```diff
@@ -180,7 +180,7 @@ def replace_static_url(original, prefix, quote, rest):
             else:
                 # if not, then assume it's courseware specific content and then look in the
                 # Mongo-backed database
-                url = StaticContent.convert_legacy_static_url_with_course_id(rest, course_id)
+                url = StaticContent.get_canonicalized_asset_path(course_id, rest)
```

Comment:
If you passed the CDN prefix as a separate param here, would that allow you to take advantage of `ConfigurationModel` -- i.e. import the config model into this file (which already uses Django), and pass the value into `StaticContent.get_canonicalized_asset_path()`?